cinnabar/tests/test_measurements.py

```diff
@@ -13,6 +14,18 @@ def test_ground():
     assert g3 == g4
 
 
+def test_Ki_to_DG():
```

Comment:
Thanks! Maybe one thing to also test here is to try out varying the temperature (that way we know it's working as intended).

If you look at the parameterize fixture (see: https://github.com/OpenFreeEnergy/openfe/blob/0e37fa53362d6e69a33960e00711a4eb35e8670c/openfe/tests/protocols/test_openmmutils.py#L25-L35) you can pass in a list of values which can then be picked up by your test.

Reply:
Added that!